app/models/category.rb

```diff
@@ -0,0 +1,12 @@
+# == Schema Information
+#
+# Table name: categories
+#
+#  id          :bigint           not null, primary key
+#  description :string
+#  name        :string
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+class Category < ApplicationRecord
+end
```

Comment:
Es buena práctica agregar una validación por cada restriccíon que tenemos en la DDBB:
```suggestion
  validates :name, presence: true
end
```